workers/vercel-edge-functions/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "vercel-edge-functions",
+  "license": "MIT",
+  "scripts": {
+    "test": "jest",
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "@prisma/client": "3.5.0-integration-data-proxy-client-improvements.6",
+    "next": "11.1.3-canary.104",
```

Comment:
no more stable version? Renovate might have problems keeping that up to date.

Reply:
```suggestion
    "next": "12.0.2",
```